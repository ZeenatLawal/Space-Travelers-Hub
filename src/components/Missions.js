import React from 'react';

const Missions = () => (
  <div>
    <h1>Missions Incoming</h1>
  </div>
);

export default Missions;
