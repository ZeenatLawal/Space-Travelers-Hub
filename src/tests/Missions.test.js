import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Missions from '../components/Missions';

it('renders correctly', () => {
  const missions = renderer
    .create(<Provider store={store}><Missions /></Provider>)
    .toJSON();
  expect(missions).toMatchSnapshot();
});
