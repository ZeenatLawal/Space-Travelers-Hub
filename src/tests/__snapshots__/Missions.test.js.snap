// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mt-4 container"
>
  <table
    className="table table-striped table-bordered"
  >
    <thead>
      <tr>
        <th>
          Mission
        </th>
        <th>
          Description
        </th>
        <th>
          Status
        </th>
        <th>
           
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
